cfgov/jinja2/v1/_includes/organisms/header.html

```diff
@@ -95,6 +95,11 @@
         </div>
 
     </div>
+    <script>
+      /* TODO: scope no-js to individual molecules/organisms. This is a hack to fix no-js on the on-demand header, while also allowing the menu to be scoped to o-header for on-demand pages. */
```

Comment:
Like before, I'd like to make it more obvious we should remove it as soon as possible.